tests/e2e/default-mobile/old-navigation/old-navigation.wdio-spec.js

```diff
@@ -42,7 +42,8 @@ describe('Old Navigation', () => {
     permissions.can_view_old_navigation = offlineUser.roles;
     await utils.updateSettings({ tasks, permissions }, true);
 
-    await loginPage.login(offlineUser);
+    await loginPage.login({username: offlineUser.username, password: offlineUser.password, loadPage: false});
```

Comment:
Why did you put the loadPage as `false`? 🤔

Reply:
After the login action, this login function [waits for an element](https://github.com/medic/cht-core/blame/68cab433ccb537f19aca644ca87ab207b9d4e563/tests/page-objects/default/common/common.wdio.page.js#L304) to verify that the webapp is loaded. That element is different in the old nav from the new nav. Passing `false` skips [the `wait` that works in new nav](https://github.com/medic/cht-core/blob/128c1161aa22239ad90f5e224d0bef80e9d68a19/tests/page-objects/default/common/common.wdio.page.js#L332), and then we can use [a wait for the old nav](https://github.com/medic/cht-core/blob/128c1161aa22239ad90f5e224d0bef80e9d68a19/tests/page-objects/default/old-navigation/old-navigation.wdio.page.js#L109).